.changeset/thick-pillows-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-node': patch
```

Comment:
Think we should add the `@backstage/integrations` package here too

Reply:
See commit: df16f5f722a9